.changes/nextrelease/mockhandlerExcpetionFix.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The whole thing should be in square brackets, that's why this is failing travis- the formatting is very finicky because it is used to make our versioning files